src/operation/get-parameters-as-json-schema.ts

```diff
@@ -392,13 +427,9 @@ export default function getParametersAsJSONSchema(
        *
        * @todo
        */
-      if (components) {
-        // We should only include components if we've got circular refs or we have discriminator
-        // mapping refs (we want to include them).
-        if (hasCircularRefs || (hasDiscriminatorMappingRefs && opts.includeDiscriminatorMappingRefs)) {
-          // Fixing typing and confused version mismatches
-          (group.schema.components as ComponentsObject) = components;
-        }
```

Comment:
I was so close to fixing this performance issue when I added line 398. If only I had gone one step further by not running `transformComponents()`. Ah well.